luxonis_train/loaders/utils.py

```diff
@@ -38,6 +38,7 @@ def collate_fn(
             TaskType.CLASSIFICATION,
             TaskType.SEGMENTATION,
             TaskType.ARRAY,
+            TaskType.LABEL,
```

Comment:
We'll wait for the new luxonis-ml aug refactor to figure what is the best way to handle this. CC: @kozlov721 on this also (can sync offline on the requirements)